maui-toolkit/Bottom-Sheet/BottomSheet-Customization.md

```diff
@@ -195,65 +166,66 @@ Content = bottomSheet;
 {% endhighlight %}
 {% endtabs %}
 
-## Content Width Mode
-The `ContentWidthMode` property allows you to adjust the width of the `BottomSheet`. The default value is `Full`. This property accepts the following values:
+## Adjust the Height
+### FullExpandedRatio
 
-* **Full** - The sheet will occupy the entire screen width.
-* **Custom** - The sheet will adjust its width based on the value set in the `BottomSheetContentWidth` property.
+The `FullExpandedRatio` property adjusts the height of the `BottomSheet` when it is in the `FullExpanded` state. The default value is `1`. You can set a value between 0.1 and 1 to adjust the height. This property works only when the sheet's `State` is `FullExpanded`.
```

Comment:
The `FullExpandedRatio` property adjusts the height of the `BottomSheet` when it is in the `FullExpanded` state. The default value is `1`. You can set a value between 0.1 and 1 to adjust the height.

Reply:
Updated the content